src/insert-style.js

```diff
@@ -1,9 +1,9 @@
 export function getStyle(elm) {
```

Comment:
To clarify, I actually like the way you've refactored it, but I just want to understand if this was a necessary change to facilitate the tests or what exactly?